src/index.html

```diff
@@ -1,17 +1,158 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label>Surname:
+            <input
+              type="text"
+              name="surname"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label>Name:
+            <input
+              type="text"
+              name="name"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label>How old are You?
+            <input
+              type="number"
+              name="age"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label>Full date of birth:
+            <input type="date" name="dateOfBirth">
+          </label>
+        </div>
+        <div class="form-field">
+          <label class="form-field">I accept the term of the agreement
+            <input type="checkbox" name="agreement">
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
```

Comment:
```suggestion
      </fieldset>
      
      <fieldset class="fieldset">
```
left space between blocks of code for better readability